containers/rocm/.shellcheckrc

```diff
@@ -0,0 +1 @@
+disable=SC2086
```

Comment:
#980 fixes the problem. I can remove the ignore file after I have rebased my PR.